cadvisor.yaml

```diff
@@ -30,6 +30,14 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package for cadvisor
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/run
+          ln -sf /var/run "${{targets.contextdir}}"/run
```

Comment:
we can consider adding a stat or readlink -f based test here if it's required so that we don't miss this in b/w updates.